lib/database/player-game.php

```diff
@@ -0,0 +1,446 @@
+<?php
+
+use RA\ActivityType;
+
+function testFullyCompletedGame($gameID, $user, $isHardcore, $postMastery): array
+{
+    sanitize_sql_inputs($gameID, $user, $isHardcore);
+    settype($isHardcore, 'integer');
+
+    $query = "SELECT COUNT(ach.ID) AS NumAch, COUNT(aw.AchievementID) AS NumAwarded FROM Achievements AS ach
+              LEFT JOIN Awarded AS aw ON aw.AchievementID = ach.ID AND aw.User = '$user' AND aw.HardcoreMode = $isHardcore 
+              WHERE ach.GameID = $gameID AND ach.Flags = 3 ";
```

Comment:
more opportunities for `AchievementType::CORE` in this file. probably should spin up a separate PR